icu4c/source/test/intltest/usettest.cpp

```diff
@@ -4283,35 +4279,7 @@ void UnicodeSetTest::TestCodePointIterator() {
     // codePoints() returns USetCodePoints for which explicit APIs are tested via USet.
 }
 
-void UnicodeSetTest::TestUSetCodePointIterator() {
-    IcuTestErrorCode errorCode(*this, "TestUSetCodePointIterator");
-    using U_HEADER_NESTED_NAMESPACE::USetCodePoints;
-    LocalUSetPointer uset(uset_openPattern(u"[abcçカ🚴]", -1, errorCode));
-    UnicodeString result;
-    for (UChar32 c : USetCodePoints(uset.getAlias())) {
-        // printf("uset.codePoint U+%04lx\n", (long)c);
-        result.append(u' ').append(c);
-    }
-    assertEquals(WHERE, u" a b c ç カ 🚴", result);
-
-    USetCodePoints range1(uset.getAlias());
-    auto range2(range1);  // copy constructor
-    auto iter = range1.begin();
-    auto limit = range2.end();
-    // operator* with pre- and post-increment
-    assertEquals(WHERE, u'a', *iter);
-    ++iter;
-    assertEquals(WHERE, u'b', *iter);
-    assertEquals(WHERE, u'c', *++iter);
-    auto iter2(iter);  // copy constructor
-    assertEquals(WHERE, u'c', *iter2++);
-    assertEquals(WHERE, u'ç', *iter2++);
-    assertEquals(WHERE, u'カ', *iter2);
-    assertTrue(WHERE, ++iter2 != limit);
-    auto iter3(iter2++);
-    assertEquals(WHERE, U'🚴', *iter3);
-    assertTrue(WHERE, iter2 == limit);
-}
+// TestUSetCodePointIterator moved to headeronlytest.cpp
```

Comment:
I doubt that future readers will find such comments helpful, most of them will be unlikely to know that these test cases ever where here in the first place.

Reply:
ok done